api/types/configs.go

```diff
@@ -0,0 +1,27 @@
+package types
+
+// configs holds structs used for internal communication between the
+// frontend (such as an http server) and the backend (such as the
+// docker daemon).
+
+import (
+	"github.com/docker/docker/runconfig"
+)
+
+// ContainerRmConfig is a holder for passing in runtime config.
```

Comment:
Not sure what this comment means, but doesn't seem to match the usage of this struct.